### ISSUE TITLE
Category Notes - Reveal on hover 

### ISSUE THREAD

**Issue by Kidglove57** [actualbudget/releases#131]
When allocating budgeted amounts each month, I rely on my category notes which are helpful stored in the budget screen. At present I need to click into each note individually to check the amount. It would be SO helpful if my notes could be revealed by just hovering the mouse over the notes icon instead. See YNAB4 purely as an example of where this has been done before.

**Comment by bpaulien** [comment]
There was a small discussion on the "discussions" tab regarding this.  Just linking that here, for further reference.

https://github.com/actualbudget/releases/discussions/79#discussion-3690969

**Comment by bpaulien** [comment]
> Thanks for linking! So if I read it right this my posted issue just goes one step beyond the original discussion and asks for the full content of the note to be revealed on hover?

Yeah, basically.  I'm not sure exactly what the difference is between the "issues" and "discussions" tabs.  I think the dev looks at both.  I think "issues" is more like bug reports, and "discussion" is feature requests.  But the lines blur, and I'm not sure it's always consistent.
